packages/amplify-e2e-tests/src/__tests__/migration/node.function.test.ts

```diff
@@ -3,7 +3,7 @@ import * as fs from 'fs-extra';
 import _ from 'lodash';
 import {
   nspawn as spawn,
-  getCLIPath,
+  getCLIPath, getScriptRunnerPath,
```

Comment:
nit: move to new line

```suggestion
  getCLIPath,
  getScriptRunnerPath,
```